js_modules/dagster-ui/packages/ui-core/src/assets/AssetsOverview.tsx

```diff
@@ -0,0 +1,333 @@
+    }),
+  );
+  if (hour < 12) {
+    return 'Good morning';
+  } else if (hour < 18) {
+    return 'Good afternoon';
+  } else {
+    return 'Good evening';
+  }
+}
+
+const CountForAssetType = ({children, assetsCount}: AssetOverviewCategoryProps) => {
+  return (
+    <Box
+      flex={{direction: 'row', justifyContent: 'space-between'}}
+      style={{width: 'calc(33% - 16px)'}}
+    >
+      <div>{children}</div>
+      <AssetCount>{assetsCount} assets</AssetCount>
+    </Box>
+  );
+};
+
+const SectionHeader = ({sectionName}: {sectionName: string}) => {
+  return (
+    <Box
+      flex={{alignItems: 'center', justifyContent: 'space-between'}}
+      padding={{horizontal: 24, vertical: 8}}
+      border="top-and-bottom"
+    >
+      <SectionName>{sectionName}</SectionName>
+    </Box>
+  );
+};
+
+const SectionBody = ({children}: {children: React.ReactNode}) => {
+  return (
+    <Box
+      padding={{horizontal: 24, vertical: 16}}
+      flex={{wrap: 'wrap'}}
+      style={{rowGap: '14px', columnGap: '24px'}}
+    >
+      {children}
+    </Box>
+  );
+};
+
+const linkToAssetGraphGroup = (groupMetadata: GroupMetadata) => {
+  return `/asset-groups?${qs.stringify({groups: JSON.stringify([groupMetadata])})}`;
+};
+
+const linkToAssetGraphComputeKind = (computeKind: string) => {
+  return `/asset-groups?${qs.stringify({
+    computeKindTags: JSON.stringify([computeKind]),
+  })}`;
+};
+
+const linkToCodeLocation = (repoAddress: RepoAddress) => {
+  return `/locations/${repoAddressAsURLString(repoAddress)}/assets`;
+};
+
+export const AssetsOverview = ({viewerName}: {viewerName?: string}) => {
+  useTrackPageView();
+
+  const params = useParams();
+  const currentPath: string[] = ((params as any)['0'] || '')
+    .split('/')
+    .filter((x: string) => x)
+    .map(decodeURIComponent);
+
+  useDocumentTitle('Assets');
+
+  const assetsQuery = useQuery<AssetCatalogTableQuery, AssetCatalogTableQueryVariables>(
+    ASSET_CATALOG_TABLE_QUERY,
+    {
+      notifyOnNetworkStatusChange: true,
+    },
+  );
+  const assetCountBySection = buildAssetCountBySection(
+    assetsQuery.data?.assetsOrError.__typename === 'AssetConnection'
+      ? assetsQuery.data.assetsOrError.nodes
+      : [],
+  );
+  const {UserDisplay} = useLaunchPadHooks();
+  const {
+    timezone: [timezone],
+  } = useContext(TimeContext);
+
+  if (assetsQuery.loading) {
+    return (
+      <Page>
+        <AssetPageHeader assetKey={{path: currentPath}} />
+        <Box flex={{direction: 'row', justifyContent: 'center'}} style={{paddingTop: '100px'}}>
+          <Box flex={{direction: 'row', alignItems: 'center', gap: 16}}>
+            <Spinner purpose="body-text" />
+            <div style={{color: Colors.textLight()}}>Loading assets…</div>
+          </Box>
+        </Box>
+      </Page>
+    );
+  }
+
+  return (
+    <>
+      <AssetPageHeader
+        assetKey={{path: currentPath}}
+        right={
+          <Box flex={{gap: 12, alignItems: 'center'}}>
```

Comment:
With only one item in the flex container, the `gap` shouldn't be necessary here.